src/index.css

```diff
@@ -16,34 +16,25 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
-}
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
Great job @didierganthier 🤝

- [ ] [Optional] You may add a margin-bottom to your design so as to create some space between the bottom of the page and the Add New Book form.

Reply:
Thank you @1GORDON 😊